content/learn/08.contributions/03.arduino-creating-library-guide/arduino-creating-library-guide.md

```diff
@@ -228,7 +244,7 @@ Morse morse(13);
 
 When this line gets executed (which actually happens even before the `setup()` function), the constructor for the `Morse` class will be called, and passed the argument you've given here (in this case, just 13).
 
-Notice that our `setup()` is now empty; that's because the call to `pinMode()` happens inside the library (when the instance is constructed).
+Notice that our `setup()` now has a call to `morse.begin()` which will make a call to `pinMode()` inside the library. It is best practice to have hardware initialization calls like `pinMode()` take place in `setup()` which is why we created the `begin()` function.
```

Comment:
```suggestion
Notice that our `setup()` now has a call to `morse.begin()` which configure the pin that was set in the constructor.
```

This section of the tutorial is focused on the sketch code, so it should not contain low level information about the library. That type of content should be in the section of the tutorial focused on the library code.

Reply:
Sold!  I would just change "configure" to "configures" for subject-verb agreement.